bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -85,7 +85,7 @@
               </ul>
             </div>
             {% if l10n_has_tag('firefox_new_other_platforms') %}
```

Comment:
No, because removing this here would expose the modal link for everyone, but then if the modal itself wasn't also translated, clicking the link would likely break?